handler/transport.go

```diff
@@ -33,8 +34,15 @@ type transport struct {
 }
 
 func (t *transport) RoundTrip(r *http.Request) (*http.Response, error) {
-	if err := t.prov.VerifyRoleToken(r.Context(), r.Header.Get(t.cfg.RoleHeader), r.Method, r.URL.Path); err != nil {
-		return nil, errors.Wrap(err, ErrMsgVerifyRoleToken)
+	if t.cfg.BypassURLPath == "" || t.cfg.BypassURLPath != r.URL.Path {
+		if err := t.prov.VerifyRoleToken(r.Context(), r.Header.Get(t.cfg.RoleHeader), r.Method, r.URL.Path); err != nil {
+			return nil, errors.Wrap(err, ErrMsgVerifyRoleToken)
+		}
+	} else {
+		// WARNING!!! Authorization is bypassed.
+		if err := glg.Debug("Authorization checking skipped on: " + r.URL.Path); err != nil {
```

Comment:
thinking if should be debug message, as it is expected behavior. The comment above is not need.

Reply:
fixed: https://github.com/yahoojapan/authorization-proxy/pull/41/files/967110a82738c05cf44e14fac9b1734d99e10eaa..1360c02583f954790106fb51f367723d4d034239